<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAddSucursal</class>
 <widget class="QWidget" name="FrmAddSucursal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sucursal</string>
  </property>
  <widget class="QGroupBox" name="grBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Información de Sucursal</string>
   </property>
   <widget class="QLabel" name="lblDpto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Departamento</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboDepartamento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>54</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblCiudad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>84</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ciudad</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboCiudad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>114</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblSucursal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre Sucursal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSucursal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>174</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>204</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dirección Sucursal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>234</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>294</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string>9999999999</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblTelefono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>264</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teléfono Sucursal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblObservaciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>324</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Observaciones</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtObservaciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>360</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
